src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -64,6 +64,10 @@ if(NOT (AARCH64 OR ARM))
     list(APPEND EXCLUDE_PATHS ${CMAKE_CURRENT_SOURCE_DIR}/src/transformations/cpu_opset/arm/*)
 endif()
 
+if (NOT ENABLE_MLAS)
```

Comment:
`ENABLE_MLAS` should be explicitly enabled by default, isn't it?

Reply:
Current, it's not enabled by default, we could open it now